src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/pipeline_management/create_repository.py

```diff
@@ -67,6 +68,34 @@ def lambda_handler(event, _):
         .get("properties", {})
         .get("repository", {})
     )
+
+    # Resolve code_account_id in case it is not set
+    # Evaluate as follows: 
+    # If source_account_id not set, we have to set it as follows:
+    #   - set via default_scm_codecommit_account_id (if exists)
+    #   - or set via ADF_DEPLOYMENT_ACCOUNT_ID
+    try:
+        default_scm_codecommit_account_id = parameter_store.fetch_parameter(
+            "/adf/scm/default-scm-codecommit-account-id"
+        )
+    except ParameterNotFoundError:
+        default_scm_codecommit_account_id = DEPLOYMENT_ACCOUNT_ID
+    if not code_account_id:
+        print("account_id not found in source_props - recreate it!")
```

Comment:
Nit: Don't use print, use a logger

Reply:
Also, no need to yell, can we make this message more.... relevant?

Reply:
Added more verbose comment